pkg/api/types.go

```diff
@@ -1245,6 +1247,14 @@ func (m *MasterProfile) IsUbuntuNonVHD() bool {
 	return m.IsUbuntu() && !m.IsVHDDistro()
 }
 
+// IsAuditDEnabled checks if auditd is enabled
+func (m *MasterProfile) IsAuditDEnabled() bool {
```

Comment:
we don't really need this function, `to.Bool` does the nil check for you. You can replace every use of `IsAuditDEnabled()` by `to.Bool(m.AuditDEnabled)`